src/Control/Lens/Internal/Zoom.hs

```diff
@@ -291,7 +291,7 @@ instance (Apply m, Semigroup r) => Semigroup (Effect m r a) where
   Effect ma <> Effect mb = Effect (liftF2 (<>) ma mb)
   {-# INLINE (<>) #-}
 
-instance (Monad m, Monoid r) => Monoid (Effect m r a) where
+instance (Apply m, Monad m, Monoid r) => Monoid (Effect m r a) where
```

Comment:
I never used `Zoom` functionality, so I cannot comment on this change. Is here better to add `Monad` to the `Semigroup`, not sure.

Reply:
If someone knows any public package where `magnify` or such are used, I'm keen to look and try how this affects them.